### ISSUE TITLE
<code> WET style does not pass colour contrast threshold

### ISSUE THREAD

**Issue by campjef** [wet-boew/wet-boew#7810]
By default, the <code> tag formats text to look similar to a Courier font. In WET, This style has be changed to a reddish colour on a pinkish background:

code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}

While the colours as defined in the css pass, the rendered colours do not, The rendered version, when tested with the Coulour Contrast Analyser, is actually pixels of different colours. The result is that the contrast threshold is met only when you can find a dark pixel, but the rest of the time the contrast threshold is not met and fails badly. This style needs to be updated in order to pass the WCAG 2.0 colour contrast threshold of 4.5:1 when rendered in a browser.

**Comment by campjef** [comment]
Not sure why but the tag I mentioned did not show up. This tag is the **code** tag.

**Comment by LaurentGoderre** [comment]
@campjef I updated your comment, to add html you need to wrap it with ` for inline markup and three of them for block html (```)

**Comment by LaurentGoderre** [comment]
Not sure that we must meet the contrast in all of the aliasing. @mysticjeff @SBleeks @dfarough what do you guys think?

In any case, if we need to update it `#A60417` seems close and have better contrast.

**Comment by SBleeks** [comment]
I believe that when rendered, the font foreground and background colouring scheme changes. So, for example, the pink font is almost washed out and in turn doesn’t meet colour contrast requirements.

**Comment by nschonni** [comment]
The styling for those blocks depends on https://github.com/google/code-prettify. I don't think it's been updated in some time so there may also be an upstream fix we're missing at this point.
